platforms/org.eclipse.mita.platform.arduinouno/ARD_SDK/ARD_Retcode.h

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
This is a bit too close to the XDK to my taste. Potential implementors of a platform might think that it's a Mita convention to call return values/exception code types `Retcode_T` in Mita. We do not have such a convention and I am not sure we'd want to introduce one. Further, if the main purpose of the Arduino platform is to serve as an example for others, it would make sense to show some variety in how things could be done rather than replicating the already existing XDK platform.

There are two options:

- Rename`retcode` to something like `exception`
- Do away with a custom exception definition altogether and use an `stdint` type instead

Reply:
renamed it to Exception_T and exception with error code STATUS_OK